Scenes/Enemies/Enemy1.gd

```diff
@@ -52,6 +53,13 @@ func take_damage(damage):
 		emit_signal("defeated")
 		queue_free()
 
+func _on_Area2D_body_entered(collider):
+	if "Player" in collider.get_groups():
```

Comment:
A cleaner approach is to use `class_name Player` in the Spaceship script, so that here you can check with 

```gd
if collider is Player:
   print("I'm definitely the player!")
```

Reply:
https://docs.godotengine.org/en/3.1/getting_started/scripting/gdscript/static_typing.html#a-brief-look-at-static-typing